### PR TITLE
feat(otel): optimiz the option section in the Hertz OTEL doc

### PR DESCRIPTION
**Pull request by CoderPoet**
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en: optimiz the option section in the Hertz OTEL doc
zh(optional): 优化 hertz otel 文档的 option 章节

#### (Optional) Which issue(s) this PR fixes:

https://github.com/hertz-contrib/obs-opentelemetry/issues/33

### CODE REVIEW

File: content/zh/docs/hertz/tutorials/observability/open-telemetry.md [hunk 1]
```diff
@@ -142,19 +142,21 @@ func main() {
...
+| WithEnableMetrics         | 是否启用 `metrics`                     |
 | WithTextMapPropagator     | 设置 `propagation.TextMapPropagator` |
-| WithResourceDetector      | 配置 `resource.Detector`              |
-| WithHeaders               | 配置导出 telemetry 数据的 gRPC 请求头         |
-| WithInsecure              | 配置是否对导出的 gRPC 客户端使用安全认证             |
+| WithResourceDetector      | 配置 `resource.Detector`             |
+| WithHeaders               | 配置导出 telemetry 数据的 gRPC 请求头        |
+| WithInsecure              | 配置是否对导出的 gRPC 客户端使用安全认证            |
+| WithClientHttpRouteFormatter              | 配置客户端指标 `http_route` 维度自定义格式化取值函数  |
+| WithServerHttpRouteFormatter              | 配置服务端端指标 `http_route` 维度自定义格式化取值函数 |
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/observability/open-telemetry.md:159]: 加英文

File: content/zh/docs/hertz/tutorials/observability/open-telemetry.md [hunk 2]
```diff
@@ -142,19 +142,21 @@ func main() {
...
-| WithEnableMetrics         | 是否启用 `metrics`                      |
+| 函数名                       | 描述                                 |
+|---------------------------|------------------------------------|
+| WithServiceName           | 配置 `service.name` 的资源属性            |
+| WithDeploymentEnvironment | 配置`deployment.environment`资源属性     |
+| WithServiceNamespace      | 配置了`service.namespace`资源属性         |
+| WithResourceAttributes    | 配置资源属性                             |
+| WithResource              | 配置资源 (`resource.Resource`)         |
+| WithEnableTracing         | 是否启用 `tracing`                     |
+| WithEnableMetrics         | 是否启用 `metrics`                     |
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/observability/open-telemetry.md:153]: 这里也有一些 otion 是在 tracing 目录下的 https://github.com/hertz-contrib/obs-opentelemetry/blob/main/tracing/options.go

Comment (CoderPoet) [reply content/zh/docs/hertz/tutorials/observability/open-telemetry.md:164]: > 这里也有一些 option 是在 tracing 目录下的 https://github.com/hertz-contrib/obs-opentelemetry/blob/main/tracing/options.go

嗯嗯 改了下，现在单独拆了一个 tracing options 的章节

File: content/zh/docs/hertz/tutorials/observability/open-telemetry.md (hunk 1 above)
Comment (CoderPoet) [reply content/zh/docs/hertz/tutorials/observability/open-telemetry.md]: done

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**